pkg/mapper/statefulset_to_etcd.go

```diff
@@ -34,7 +35,7 @@ type statefulSetToEtcdMapper struct {
 	cl  client.Client
 }
 
-func (m *statefulSetToEtcdMapper) Map(obj client.Object) []reconcile.Request {
+func (m *statefulSetToEtcdMapper) Map(ctx context.Context, log logr.Logger, reader client.Reader, obj client.Object) []reconcile.Request {
```

Comment:
These newly added parameters are unused in this function

Reply:
I have added these parameters since they are needed for the new Mapper interface in order not to change the StatefulSetToEtcd() function. The Mapper interface was changed [in this PR](https://github.com/gardener/gardener/pull/6358).